infra/azure/main.bicep

```diff
@@ -143,11 +145,18 @@ module cosmosModule 'modules/cosmos.bicep' = {
   }
 }
 
-// Module: Azure OpenAI (Placeholder for PR #4)
-// module openaiModule 'modules/openai.bicep' = {
-//   name: 'openaiDeployment'
-//   ...
-// }
+// Module: Azure OpenAI
+module openaiModule 'modules/openai.bicep' = if (deployAzureOpenAI) {
+  name: 'openaiDeployment'
+  params: {
+    location: location
+    accountName: openaiAccountName
+    sku: azureOpenAISku
+    enablePublicNetworkAccess: environment == 'dev'  // Disable for non-dev; use Private Link in staging/prod
+    networkDefaultAction: environment == 'dev' ? 'Allow' : 'Deny'
+    tags: tags
+  }
```

Comment:
Created parameters.staging.json so the PR description matches the repo; staging configuration now lives beside dev/prod files.